parla/task_runtime.py

```diff
@@ -428,84 +422,84 @@ def is_assigned(self):
             return self._assigned
 
     @property
-    def predecessors(self) -> Tuple["Task"]:
-        return tuple(self._predecessors)
-
-    @predecessors.setter
-    def predecessors(self, predecessors: Collection['Task']):
-        self._predecessors: List[Task] = list(predecessors)
-        self._num_blocking_predecessors: int = len(predecessors)
-        for dep in predecessors:
+    def dependencies(self) -> Tuple["Task"]:
+        return tuple(self._dependencies)
+
+    @dependencies.setter
+    def dependencies(self, dependencies: Collection['Task']):
+        self._dependencies: List[Task] = list(dependencies)
```

Comment:
Why is dependencies a collection and not just a list from the start?

Reply:
Same question; this exists from 2yr olds.

Reply:
Also less importantly this type hint is not strictly accurate, as this Collection can also have taskIDs (as mentioned in the comment near there)